src/index.ts

```diff
@@ -0,0 +1,7 @@
+export default {
+    isValid(bic: string): boolean {
+        const bicRegex = /^[A-Z]{6}[A-Z0-9]{2}([A-Z0-9]{3})?$/;
```

Comment:
I'd rather keep the regex declaration outside of the function. Could you move it up?